dataflow/src/main/java/org/checkerframework/dataflow/cfg/StringCFGVisualizer.java

```diff
@@ -73,15 +65,25 @@ public String visualizeBlock(Block bb, @Nullable Analysis<V, S, T> analysis) {
 
     @Override
     public String visualizeSpecialBlock(SpecialBlock sbb) {
-        return super.visualizeSpecialBlockHelper(sbb, lineSeparator);
+        switch (sbb.getSpecialType()) {
+            case ENTRY:
+                return "<entry>" + lineSeparator;
```

Comment:
Can you add a comment why the lineSeparator is only needed in this case?
Wasn't it nicer before with the `Helper` method that reduced duplication?

Reply:
I have pushed a better solution... The previous change has been reverted.